packages/account-v2/src/modules/IDVForm/__tests__/idv-form.spec.tsx

```diff
@@ -29,4 +68,81 @@ describe('IDVForm', () => {
     it('should throw error when IDVform is not wrapped with Formik', () => {
         expect(() => render(<IDVForm selectedCountry={{}} />)).toThrowError();
     });
+
+    it('Should change the document type value when document type is changed', async () => {
+        renderComponent();
+        const elDocumentType = screen.getByLabelText(documentTypeLabel);
+
+        userEvent.click(elDocumentType);
+        expect(await screen.findByText('Document 1')).toBeInTheDocument();
+
+        userEvent.tab();
+        await waitFor(() => {
+            expect(screen.queryByText('Document 2')).not.toBeInTheDocument();
+        });
+    });
+
+    it('should render the hint messages for the selected document', async () => {
+        renderComponent();
+        const elDocumentType = screen.getByLabelText(documentTypeLabel);
+
+        jest.spyOn(utils, 'getSelectedDocumentConfigData').mockReturnValue(mockDocumentConfig);
+
+        userEvent.click(elDocumentType);
+
+        const elSelectedOption = await screen.findByText('Document 1');
+
+        userEvent.click(elSelectedOption);
+
+        expect(await screen.findByText('Example: 1234567890')).toBeInTheDocument();
+    });
+
+    it("Should hide document number field when 'I dont have any of these is chosen'", async () => {
+        const newProps = {
+            ...mockProps,
+            allowDefaultValue: true,
+        };
+
+        renderComponent(newProps);
+
+        const elDocumentTypeInput = screen.getByLabelText(documentTypeLabel);
+        const elDocumentNumberInput = screen.getByText('Enter your document number');
+
+        expect(elDocumentTypeInput).toBeVisible();
```

Comment:
can we use `toBeInTheDocument()` here instead of `toBeVisible`?

Reply:
We can. I came across this - https://stackoverflow.com/questions/69852702/what-is-the-difference-between-tobeinthedocument-and-tobevisible